README.md

```diff
@@ -738,7 +791,18 @@ module.exports = {
 
 #### `transformAll`
 
-Type: `Function`
+Type:
+
+```ts
+type transformAll = (
+  data: {
+    data: Buffer;
+    sourceFilename: string;
+    absoluteFilename: string;
+  }[]
+) => any;
```

Comment:
Should not be `any`

Reply:
- https://github.com/webpack-contrib/copy-webpack-plugin/blob/9b5a314b71e9ab16ad3ada0816620aa75d18ca47/types/index.d.ts#L70

🤔  Am I looking at the wrong type then 😕

Reply:
We should add `@returns` to types here too, looks like bug on our side

Reply:
- https://github.com/webpack-contrib/copy-webpack-plugin/blob/9b5a314b71e9ab16ad3ada0816620aa75d18ca47/types/index.d.ts#L253-L259

I thought this returned any